packages/display/src/DisplayObject.ts

```diff
@@ -447,6 +448,7 @@ export abstract class DisplayObject extends utils.EventEmitter<DisplayObjectEven
     /** Updates the object transform for rendering. TODO - Optimization pass! */
     updateTransform(): void
     {
+        if (!this.transform) return;
         this._boundsID++;
 
         this.transform.updateTransform(this.parent.transform);
```

Comment:
What about using optional-chaining instead of an if-statement to bail-out?

```suggestion
        this.transform?.updateTransform(this.parent.transform);
```

Reply:
Yes that looks cleaner.